srcpkgs/chroot-bash/template

```diff
@@ -6,7 +6,9 @@ version="${_bash_distver}.${_bash_patchlevel}"
 revision=1
 wrksrc="bash-${_bash_distver}"
 build_style=gnu-configure
-configure_args="--without-bash-malloc --without-curses --without-installed-readline --disable-nls"
+# need 'bash_cv_termcap_lib=gnutermcap' in order to force bash to use the
+# bundled termcap library when bootstrapping from source on a foreign system
+configure_args="--without-bash-malloc --without-curses --without-installed-readline --disable-nls bash_cv_termcap_lib=gnutermcap"
```

Comment:
please cut at 80 cols and bump rev

Reply:
Is bumping rev necessary? It does *not* change the resulting binary for our existing chroot-bash packages, as those are built in an environment that forces them to use the bundled one. This only changes things for those who are bootstrapping on foreign systems.